packages/harmony/src/components/comments/CommentText/CommentText.stories.tsx

```diff
@@ -28,8 +27,8 @@ export const Default: Story = {
       <CommentText>This is a test with some short text.</CommentText>
       <Divider />
       <CommentText>
-        This is a test to see <TextLink variant='visible'>1:24</TextLink> how
-        the new component for the comment body handles text of different sizes.
+        This is a test to see how the new component for the comment body handles
```

Comment:
we wanted to remove textlink here?

Reply:
Yes because CommentText only supports a `string` child now. This is needed so it can be passed to UserGeneratedText